### ISSUE TITLE
NativeWindow X/Y Properties sometimes don't do anything on Ubuntu

### ISSUE THREAD

**Issue by xanather** [opentk/opentk#233]
Simple issue, changing the X/Y properties on with NativeWindow does nothing

I'm creating quite a few issues here ill see if I am able to investigate some of them myself

**Comment by xanather** [comment]
GameWindow seems to handle it fine, must be something to do with my code. Ill close this for now but reopen if its a issue with NativeWindow

**Comment by xanather** [comment]
I was wrong, GameWindow has the issue too. On Linux/Ubuntu only sometimes does the actual window react to changing the X/Y properties.

As a work around this seems to work (this is code for centering windows):

```C#
Rectangle deviceRectangle = primaryDisplayBounds; //Bounds of any display here
int locX = deviceRectangle.X + deviceRectangle.Width / 2 - window.Width / 2;
int locY = deviceRectangle.Y + deviceRectangle.Height / 2 - window.Height / 2;
while (window.X != locX || window.Y != locY)
{
window.X = locX;
window.Y = locY;
window.ProcessEvents();
}
```

Not optimal, still seems like an issue with the implementation.

**Comment by Frassle** [comment]
Is this a problem that setting doesn't effect at all or doesn't effect straight away (that is, it sets just after a frame delay)

**Comment by xanather** [comment]
Well, after setting X/Y nothing happens (almost all of the time), unless I loop through and force it until its the correct value.

**Comment by Frassle** [comment]
OK I'm getting a spare ubuntu machine this weekend so I'll see if I can repro this.

**Comment by xanather** [comment]
This was a strange issue I posted. In a lot of scenario's it works fine. When I posted this issue I was trying to use the default ubuntu graphics drivers and have a OpenGL 3.0 context, which seemed to make the window very buggy.

Right now I am running with a OpenGL 2.1 context and using nvidia graphics drivers, I am unable to test a 3.3 context for this issue because of https://github.com/opentk/opentk/issues/108 (which I think its largely not OpenTK's fault). The OpenGL 2.1 context and nvidia drivers work fine and this is not an issue there.

Not sure how GL drivers/context can mess with window management.
